dist/rpm/os-autoinst.spec

```diff
@@ -130,14 +124,9 @@ ln -s ../sbin/service %{buildroot}%{_sbindir}/rcos-autoinst-openvswitch
 export NO_BRP_STALE_LINK_ERROR=yes
 
 %check
-# disable code quality checks - not worth the time for package builds
-sed '/perlcritic/d' -i Makefile
-sed '/Perl::Critic/d' -i cpanfile
-sed '/tidy/d' -i Makefile
-
 # should work offline
 for p in $(cpanfile-dump); do rpm -q --whatprovides "perl($p)"; done
-make check test VERBOSE=1 %{make_check_args}
+make %{?_smp_mflags} VERBOSE=1 test-perl-testsuite test-installed-files test-doc
```

Comment:
I would prefer to call a more generic target here to make sure we don't forget to cover any potential new target here and only exclude explicit sub-tests with a blocklist

Reply:
I guess I could actually call ctest here directly (it allows specifying an exclude regex).

Reply:
I've added a special target so we don't have to bother with ctest-syntax at this level. This target internally uses an exclude-pattern.